website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -400,6 +421,12 @@ models:
       [on_configuration_change](/reference/resource-configs/on_configuration_change): apply | continue | fail
       [target_lag](#target-lag): downstream | <time-delta>
       [snowflake_warehouse](#configuring-virtual-warehouses): <warehouse-name>
+<VersionBlock firstVersion="1.9">
+
+      [refresh_mode](#refresh-mode): AUTO | FULL | INCREMENTAL
```

Comment:
Looks like the version blocks don't work inside of the code blocks. Will change this to a text warning rather than version blocking the whole tab setup